.github/workflows/build_main_documentation.yml

```diff
@@ -95,15 +99,29 @@ jobs:
           cp doc-build/${{ inputs.package }}/_versions.yml build_dir/${{ inputs.package }}
 
       - name: Make documentation
+        shell: bash
         env:
           NODE_OPTIONS: --max-old-space-size=6656
         run: |
           echo "doc_folder has been set to ${{ env.doc_folder }}"
 
-          if [ -z "${{ inputs.notebook_folder }}" ]
+          if [ -z "${{ inputs.notebook_folder }}" ];
```

Comment:
Now that I've set `shell: bash` we need to explicitly add the comma in the if statement